scripts/helper_functions.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Update version of all files
+
+let SCRIPT_LOCATION=$(dirname -- "$(readlink -f -- "${BASH_SOURCE[0]}")")
+
+function updateVersion() {
+  if [ $# -neq 3 ]; then
+    echo "Wrong number of arguments!"
+    return 2
+  fi
+
+  echo "Updating task.json..."
+  updateTask $1 $2 $3
+
+  VERSION="${1}.${2}.${3}"
+
+  echo "Updating vss-extension.json..."
+  updateVSSExtension $VERSION
+
+  echo "Updating package.json..."
+  updatePackage $VERSION
+
+  echo "Versions are up to date!"
+
+  return 0
+}
+
+function updateTask() {
+  cd $SCRIPT_LOCATION/../publishjgivenreport
```

Comment:
I'd refrain from using cd in a script. You never know where you're going to end up. Especially if you interrupt.
So I prefer to address files via absolute path.

Reply:
Point taken.